04-spatial-operations.Rmd

```diff
@@ -404,7 +413,7 @@ joined = st_join(x = asia, y = urb) %>%
   na.omit()
 ```
 
-
+<!-- The yellow point is hardly recognizable, maybe use also the border parameter? -->
```

Comment:
The yellow point is hardly recognizable in the figure (Tokio), maybe use also the border parameter?

Reply:
Good point. Sorted with a rainbow pal. One question: how to make other pallettes (e.g. `viridis::*` ones) work?

Reply:
Nudge.